postgraphile/website/versioned_docs/version-4/running-postgraphile-in-docker.md

```diff
@@ -461,7 +469,7 @@ In the same folder `custom-plugin`, create a new file `index.js` with the
 following content.
 
 ```js
-const { makeWrapResolversPlugin } = require("graphile-utils");
+const { makeWrapResolversPlugin } = require("graphile/utils");
```

Comment:
```suggestion
const { makeWrapResolversPlugin } = require("graphile-utils");
```

In V5 it's `postgraphile/utils`; in V4 (and V5, technically, since the prior is just a redirect) it's `graphile-utils`; it's never `graphile/utils`